app/serializers/brack_serializer.rb

```diff
@@ -0,0 +1,10 @@
+require 'brackr/distance_calculator'
+
+class BrackSerializer < ActiveModel::Serializer
+  include Brackr::Distance
```

Comment:
do we need to include the entire module here. could we just use the module...

```
def distance
  Brackr::Distance.calculate_distance()
end
```

this would require you to make `calculate_distance` a "class" method, but it might be nice from an OO standpoint, not to include the entire module and just use what you need